Encountify/ViewModels/BaseViewModel.cs

```diff
@@ -53,6 +53,7 @@ public string Title
 
         public List<Location> DummyLocationList()
```

Comment:
We could probably try to put our locations in the server now and get it from there, instead of this